packages/typescript/internal/ts_project.bzl

```diff
@@ -568,6 +576,18 @@ def ts_project_macro(
             will be called with the remaining arguments.
             See the packages/typescript/test/ts_project/swc directory for an example.
 
+            When a custom transpiler is used (i.e. `transpiler != "tsc"`) then this macro expands to these targets:
+
+                - `[name]` - the default target is a `js_library` which can be included in the `deps` of downstream rules.
+                    Note that it will successfully build *even if there are typecheck failures* because the `tsc` binary
+                    is not needed to produce the default outputs.
+                    This is considered a feature, as it allows you to have a faster development mode where typechecking
+                    is not on the critical path.
+                - `[name].check` - this target will fail to build if the typechecking fails, useful for CI.
+                - `_[name]_tsc` - internal target which runs the binary from the `tsc` attribute
+                - ?? - whatever targets the custom transpiler rule/macro produces.
```

Comment:
nit: "Any additional targets the custom transpiler rule/macro produces.", the `??` seems like you forgot something